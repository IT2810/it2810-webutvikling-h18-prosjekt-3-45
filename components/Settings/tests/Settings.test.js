import React from 'react';
import renderer from 'react-test-renderer';
import { Settings } from '../index';

jest.mock('../GoalToast', () => () => 'GoalToast');

describe('Settings', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Settings />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
