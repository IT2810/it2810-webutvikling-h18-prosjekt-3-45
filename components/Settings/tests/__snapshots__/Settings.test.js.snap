// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Settings renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "flex": 1,
      "height": 1334,
    }
  }
/>
`;
